play-media-native/theme/theme.ts

```diff
@@ -14,6 +14,7 @@ export const theme = {
       lightest: "#666666",
       dark: "#0a0a0a",
       darkest: "#000000",
+      transluscent: "rgba(0, 0, 0, 0.8)",
```

Comment:
Some styles appear to have changed throughout the app 

For example 

![image](https://user-images.githubusercontent.com/96472119/214002987-46acb544-8841-4ac5-84f7-a481bd0ec37f.png)

![image](https://user-images.githubusercontent.com/96472119/214003051-a7b3eb24-f51a-4f98-b561-f52d4b0c13f1.png)